datastream-bigquery/src/main/java/com/linkedin/datastream/bigquery/BigquerySchemaRegistryClientConfig.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2020 Wayfair LLC. All rights reserved.
+ *  Licensed under the BSD 2-Clause License. See the LICENSE file in the project root for license information.
+ *  See the NOTICE file in the project root for additional information regarding copyright ownership.
+ */
+
+package com.linkedin.datastream.bigquery;
+
+/**
+ * A class that holds Schema Registry Client configuration constants.
+ */
+public class BigquerySchemaRegistryClientConfig extends io.confluent.kafka.schemaregistry.client.SchemaRegistryClientConfig {
+
+    /**
+     * A configuration to enable/disable validating schema field names when parsing Avro schemas.
+     */
+    public static final String SCHEMA_REGISTRY_PARSER_VALIDATE_FIELD_NAMES = "schema.registry.parser.validate.names";
```

Comment:
What case is this covering? Can you share an example from our environment?

Reply:
The Avro schema parser has two validation settings that can be toggled, both of which are enabled by default. The field name validation validates that field names conform to the Avro spec and do not contain illegal characters. My guess is that older versions of the Avro spec did not define rules for field names, so this field name validation option can be turned off for legacy schemas. I don't know if any of our streams have schemas with invalid field names, but I included the option so we can control this validation option if needed.